src/test/resources/testfiles/mock_event_gzip.json

```diff
@@ -0,0 +1 @@
+{"bucket":"terra-kernel-k8s-testrunner-results","contentType":"application/gzip","crc32c":"NzTz+Q\\u003d\\u003d","etag":"CM6c6KyH0vECEAE\\u003d","generation":"1625698634174030","id":"terra-kernel-k8s-testrunner-results/390ef56b-943f-43ab-92f2-73c6ca7dda22.tar.gz/1625698634174030","kind":"storage#object","md5Hash":"vYfIYE1pVfutUJgz7hbWwQ\\u003d\\u003d","mediaLink":"https://www.googleapis.com/download/storage/v1/b/terra-kernel-k8s-testrunner-results/o/390ef56b-943f-43ab-92f2-73c6ca7dda22.tar.gz?generation\\u003d1625698634174030\\u0026alt\\u003dmedia","metageneration":"1","name":"390ef56b-943f-43ab-92f2-73c6ca7dda22.tar.gz","selfLink":"https://www.googleapis.com/storage/v1/b/terra-kernel-k8s-testrunner-results/o/390ef56b-943f-43ab-92f2-73c6ca7dda22.tar.gz","size":"2133","storageClass":"STANDARD","timeCreated":"2021-07-07T22:57:14.257Z","timeStorageClassUpdated":"2021-07-07T22:57:14.257Z","updated":"2021-07-07T22:57:14.257Z","customerEncryption":{"encryptionAlgorithm":"AES256","keySha256":"6ae5555bcdd2681f1c1e5e5721b6d18a82d0aaada01d1295c22268955e9b79d4753c2f1f5f15c6e1ac80b405e366fda2f4ae26fa7390605c802b8ed7cc787c4ec8458f04c9b07fb65cea1e4344644a33bde9ce28d2eae70ff85cbebc45d6d44c7599cbebfe0b6fed5a1f275968efd29a49aedbd0fd93296f03457ebcc72b2c13"},"metadata":{"key1":"value1","key2":"value2"}}
```

Comment:
nit: pretty-print these … if a developer needs to debug a test we want to be nice and make it easy for them to read

Reply:
Reformatted json files for easy read.